docs_guide/sphinx_guide/build_docs.rst

```diff
@@ -2,6 +2,8 @@
 Build documentation locally
```

Comment:
Should this title should be changed to "How to build documentation locally" like you did for other how-tos?

Reply:
Done in 129b3e07c368669d761a978193d8f4f63f250c22